README.md

```diff
@@ -76,23 +107,33 @@ It starts with the following manual steps:
    1. Commit the changes.
 1. Create a pull request and wait for it to be approved.
 1. Merge the branch.
-1. Push a semver tag with a `v` prefix: `vX.Y.Z`.
-
-This will trigger the following automated processes:
-
-1. Two Docker images are built and pushed to a public `quay.io` registry, by the [release-master workflow](.github/workflows/release-master.yml):
-   * `quay.io/jetstack/preflight`: is pulled directly by tier 1 Jetstack Secure users, who do not have access to the Jetstack Enterprise Registry.
-   * `quay.io/jetstack/venafi-agent`: is mirrored to a public Venafi OCI registry for Venafi TLS Protect for Kubernetes users.
-
-2. The Docker images are mirrored by private Venafi CI pipelines, to:
-   * [Jetstack Enterprise Registry](https://platform.jetstack.io/documentation/installation/agent#1-obtain-oci-registry-credentials):
-     for Tier 2 Jetstack Secure users. Tier 2 grants users access to this registry.
-   * [Venafi private Registry](https://docs.venafi.cloud/vaas/k8s-components/th-guide-confg-access-to-tlspk-enterprise-components/):
-     for Tier 2 Venafi TLS Protect for Kubernetes users. Tier 2 grants users access to this registry.
-   * [Venafi public Registry](https://registry.venafi.cloud/public/venafi-images/venafi-kubernetes-agent):
-     for Tier 1 Venafi TLS Protect for Kubernetes users. Tier 1 users do not have access to the private registry. (TODO)
-
-### Helm Chart: venafi-kubernetes-agent
+1. Go to the GitHub Releases page and click "Draft a New Release".
+   - Click "Create a new tag" with the version number prefixed with `v` (e.g., `v0.1.49`).
+   - Use the title "v0.1.49",
+   - Click "Generate Release Notes"
+   - Edit the release notes to make them readable to the end-user.
+   - Click "Publish" (don't select "Draft")
```

Comment:
Thanks. When I wrote these notes originally I either overlooked the creation of a "Release" page (with release-notes) or else that wasn't a convention at the time.
My release at that time must only have resulted in a tag in GitHub which is not helpful for users.

I note that there will only be GitHub release pages for the venafi-kubernetes-agent chart **not** the jetstack-secure chart, but that's OK since jetstack-secure is considered EOL.